includes/orders/functions/actions.php

```diff
@@ -119,29 +119,34 @@ function edd_add_manual_order( $args = array() ) {
 		$status = 'complete';
 	}
 
-	// Parse date.
-	$date = sanitize_text_field( $order_data['edd-payment-date'] );
-	$hour = sanitize_text_field( $order_data['edd-payment-time-hour'] );
-
-	// Restrict to our high and low.
-	if ( $hour > 23 ) {
-		$hour = 23;
-	} elseif ( $hour < 0 ) {
-		$hour = 00;
-	}
+	$completed_date = false;
 
-	$minute = sanitize_text_field( $order_data['edd-payment-time-min'] );
+	if ( in_array( $status, edd_complete_order_status_keys() ) ) {
+		$date = sanitize_text_field( $order_data['edd-payment-date'] );
+		$hour = sanitize_text_field( $order_data['edd-payment-time-hour'] );
 
-	// Restrict to our high and low.
-	if ( $minute > 59 ) {
-		$minute = 59;
-	} elseif ( $minute < 0 ) {
-		$minute = 00;
-	}
+		// Restrict to our high and low
+		if ( $hour > 23 ) {
+			$hour = 23;
+		} elseif ( $hour < 0 ) {
+			$hour = 00;
+		}
 
-	// The date is entered in the WP timezone. We need to convert it to UTC prior to saving now.
-	$date = edd_get_utc_equivalent_date( EDD()->utils->date( $date . ' ' . $hour . ':' . $minute . ':00', edd_get_timezone_id(), false ) );
-	$date = $date->format( 'Y-m-d H:i:s' );
+		$minute = sanitize_text_field( $order_data['edd-payment-time-min'] );
+
+		// Restrict to our high and low
+		if ( $minute > 59 ) {
+			$minute = 59;
+		} elseif ( $minute < 0 ) {
+			$minute = 00;
+		}
+
+		if ( ! empty( $date ) ) {
```

Comment:
I think this is where the fatal error is coming from...it's maybe happening because of the hour/minute fields being blank. The `$hour` and `$minute` variables are both empty for me.

Reply:
Okay we can solve that, just need to set a default `hour` and `minute`

Reply:
This is fixed now and has validation to make sure when the fields are required, they are filled out.